### DISCUSSION TITLE
BOCD Example

### DISCUSSION THREAD

**Discussion by ngabrek** [IFCA-Advanced-Computing/frouros#273] [Q&A] [answered]
Hi there, 

when I'm running the example from the BOCD I get the following error message:
![BOCD](https://github.com/IFCA/frouros/assets/133040502/2e842e89-07ce-40ca-a076-deb428e0844a)
However, when I'm adding the BOCDConfig I run into another error:
![BOCDConfig](https://github.com/IFCA/frouros/assets/133040502/aa156a62-9d06-4cf1-851c-69affd7dae3a)
Does anyone know how to handle this issue?

Thank you :)

**Comment by jaime-cespedes-sisniega** [comment, marked as answer]
Hi @ngabrek 

It seems that may be a version related issue. I've run that snippet of code using [latest version](https://github.com/IFCA/frouros/releases/tag/v0.6.1) and it just works fine, so I suggest you to upgrade.

For this type of issues would be more convenient to create a [bug report](https://github.com/IFCA/frouros/issues/new?assignees=&labels=bug%2Cneeds+triage&projects=&template=bug_report.yml), so we can have a better idea of the issue.

Anyway, which version were you using? Thanks!

**Reply by ngabrek** [reply]
You're right, it was a version related issue. I was using version 0.5.1, but now that I updated the package to the latest version (0.6.1), it is working perfectly! 
Thank you!
